torch_geometric/nn/resolver.py

```diff
@@ -80,14 +122,21 @@ def normalization_resolver(query: Union[Any, str], *args, **kwargs):
 # Aggregation Resolver ########################################################
 
 
-def aggregation_resolver(query: Union[Any, str], *args, **kwargs):
+def aggregation_resolver(query: Union[Any, str], *args, reverse: bool = False,
```

Comment:
I'm not certain I like having a 'reverse' flag because it makes the logic for 'query' a bit complicated. What do you think about splitting this into `reverse_aggregation_resolver`?